Dockerfile

```diff
@@ -143,6 +97,13 @@ RUN pip3 install --requirement=/nextstrain/fauna/requirements.txt
 # accessible and importable.
 RUN pip3 install --editable /nextstrain/augur
 
+# Install additional "full" dependencies for augur.
+# TODO: these versions should be updated in augur's setup.py to work with the
+# pip install nextstrain-augur[full] approach.
```

Comment:
:+1: for updating setup.py to declare these extras and then using that extra tag here.